### PR TITLE
add imagecodecs-lite

### PR DESCRIPTION
**Pull request by ocefpaf**
First step to fix https://github.com/conda-forge/tifffile-feedstock/issues/32

### CODE REVIEW

**Comment by scopatz** [comment]
LGTM!

**Comment by ocefpaf** [comment]
@conda-forge/staged-recipes this is ready to be reviewed.

File: recipes/imagecodecs-lite/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy 1.14.*
```
Comment (chrisburr) [inline recipes/imagecodecs-lite/meta.yaml]: This doesn’t need a version specifier, the default is 1.14.

Comment (ocefpaf) [reply recipes/imagecodecs-lite/meta.yaml]: Yeah. I thought so too but see the Windows failure in the first commit.

Comment (ocefpaf) [reply recipes/imagecodecs-lite/meta.yaml]: I'll remove it in the feedstock. It is probably some bad outdated AppVeyor config. Thanks @chrisburr!

**Pull request merged**